Update Turkish Translation

mikropsoft opened the pull request.

KrishnaSSH commented:
Thanks a lot

mikropsoft commented:
@KrishnaSSH @DP-Hridayan, in the latest update, the translations appear incomplete, even though I have fully completed them. Additionally, although I only made edits on the PR, the commit shows it as a complete change over the previous file instead of just the modifications. I'm not sure if this is a GitHub bug, but when I checked, there didn't seem to be any issues with the lines.

DP-Hridayan commented:
I just realised that. Now that I think about it, I guess I mistakenly built
the release apk from the test branch (where the translations wasn't
updated) instead of master branch. I will soon release an update with
updated translations!

On Wed, 19 Mar, 2025, 3:03 am 𝗛𝗼𝗹𝗶, ***@***.***> wrote:

> @KrishnaSSH <https://github.com/KrishnaSSH> @DP-Hridayan
> <https://github.com/DP-Hridayan>, in the latest update, the translations
> appear incomplete, even though I have fully completed them. Additionally,
> although I only made edits on the PR, the commit shows it as a complete
> change over the previous file instead of just the modifications. I'm not
> sure if this is a GitHub bug, but when I checked, there didn't seem to be
> any issues with the lines.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DP-Hridayan/aShellYou/pull/195#issuecomment-2734785106>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRPG5G5AFAL4VCIO5FVWDL2VCGKHAVCNFSM6AAAAABYPSYDIGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUG44DKMJQGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: mikropsoft]*mikropsoft* left a comment (DP-Hridayan/aShellYou#195)
> <https://github.com/DP-Hridayan/aShellYou/pull/195#issuecomment-2734785106>
>
> @KrishnaSSH <https://github.com/KrishnaSSH> @DP-Hridayan
> <https://github.com/DP-Hridayan>, in the latest update, the translations
> appear incomplete, even though I have fully completed them. Additionally,
> although I only made edits on the PR, the commit shows it as a complete
> change over the previous file instead of just the modifications. I'm not
> sure if this is a GitHub bug, but when I checked, there didn't seem to be
> any issues with the lines.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DP-Hridayan/aShellYou/pull/195#issuecomment-2734785106>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRPG5G5AFAL4VCIO5FVWDL2VCGKHAVCNFSM6AAAAABYPSYDIGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUG44DKMJQGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mikropsoft commented:
> I just realised that. Now that I think about it, I guess I mistakenly built the release apk from the test branch (where the translations wasn't updated) instead of master branch. I will soon release an update with updated translations!

Nice! Yesterday, I thought I made a mistake, so I deleted the message—that's why it’s not showing up.  

By the way, I’ve updated the changelog translations. If you're planning to add any new changes related to the latest version before merging the PR, just tag me on that PR. Once I add those new lines, you can go ahead and merge it.

DP-Hridayan commented:
Thanks for translating, I have merged the PR. I will inform you before I
release the update for updated translations.

On Wed, 19 Mar, 2025, 2:23 pm 𝗛𝗼𝗹𝗶, ***@***.***> wrote:

> I just realised that. Now that I think about it, I guess I mistakenly
> built the release apk from the test branch (where the translations wasn't
> updated) instead of master branch. I will soon release an update with
> updated translations!
>
> Nice! Yesterday, I thought I made a mistake, so I deleted the
> message—that's why it’s not showing up.
>
> By the way, I’ve updated the changelog translations. If you're planning to
> add any new changes related to the latest version before merging the PR,
> just tag me on that PR. Once I add those new lines, you can go ahead and
> merge it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DP-Hridayan/aShellYou/pull/195#issuecomment-2735779323>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRPG5HKXISTOLKNP7FOV5T2VEWBFAVCNFSM6AAAAABYPSYDIGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZVG43TSMZSGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: mikropsoft]*mikropsoft* left a comment (DP-Hridayan/aShellYou#195)
> <https://github.com/DP-Hridayan/aShellYou/pull/195#issuecomment-2735779323>
>
> I just realised that. Now that I think about it, I guess I mistakenly
> built the release apk from the test branch (where the translations wasn't
> updated) instead of master branch. I will soon release an update with
> updated translations!
>
> Nice! Yesterday, I thought I made a mistake, so I deleted the
> message—that's why it’s not showing up.
>
> By the way, I’ve updated the changelog translations. If you're planning to
> add any new changes related to the latest version before merging the PR,
> just tag me on that PR. Once I add those new lines, you can go ahead and
> merge it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DP-Hridayan/aShellYou/pull/195#issuecomment-2735779323>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFRPG5HKXISTOLKNP7FOV5T2VEWBFAVCNFSM6AAAAABYPSYDIGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZVG43TSMZSGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
